frontend/apps/filemanager/filemanagermenu.lua

```diff
@@ -702,6 +702,13 @@ To:
             touchmenu_instance:updateItems()
         end,
     })
+    table.insert(self.menu_items.developer_options.sub_item_table, {
+        text = _("Dump the fontlist cache"),
+        callback = function()
+            local FontList = require("fontlist")
+            FontList:dumpFontList()
+        end,
```

Comment:
I'm not sure how large the output is - but if it's a menu item in the UI, I find it odd that it just dumps the list in the crash.log : I would expect it to (also) show it in a TextViewer - or at minima to show a InfoMessage saying "done dumping in the log, look there".

Reply:
Oh, it doesn't dump it to stdout (e.g., the crash.log), it dumps it to a dedicated file next to the actual binary cache (e.g., in cache/fotnlist/).

It's far too verbose to be of any use in a TextViewer, but I definitely could have spared an IW ;).